code-challenges.test.js

```diff
@@ -57,3 +156,18 @@ const vowelTester3 = "challenges"
 // Expected output: 2
 
 // b) Create the function that makes the test pass.
+
+// FUNCTION
+// create fuction named firstVowel
+// create a variable (vow)
+// use .length and ++ to filter with the vowel
+// use letters = a, e, i , o , u
+// if it is one of those, return vow = the exact value of vowels in the strings
```

Comment:
pseudocode description of `vow` is not explaining the purpose that it  is fulfilling. Vow will be returning the exact index or location of the vowel in the string